docs/userguides/alertrules.md

```diff
@@ -0,0 +1,122 @@
+            "ruleSource": "Alerting",
+            "fileSizeAndCount": {
+                "type$": "FILE_SIZE_AND_COUNT",
+                "fileCountGreaterThan": 2,
+                "totalSizeGreaterThanInBytes": 200,
+                "operator": "AND"
+            },
+            "fileActivityIs": {
+                "type$": "FILE_ACTIVITY",
+                "syncedToCloudService": {
+                    "type$": "SYNCED_TO_CLOUD_SERVICE",
+                    "watchBox": false,
+                    "watchBoxDrive": false,
+                    "watchDropBox": false,
+                    "watchGoogleBackupAndSync": false,
+                    "watchAppleIcLoud": false,
+                    "watchMicrosoftOneDrive": false
+                },
+                "uploadedOnRemovableMedia": true,
+                "readByBrowserOrOther": true
+            },
+            "timeWindow": 15,
+            "id": "404ff012-fa2f-4acf-ae6d-107eabf7f24c",
+            "createdAt": "2021-04-27T01:55:36.4204590Z",
+            "createdBy": "sean.cassidy@example.com",
+            "modifiedAt": "2021-09-03T01:46:13.2902310Z",
+            "modifiedBy": "sean.cassidy@example.com",
+            "isSystem": false
+        }
+    ]
+}
+```
+
+## Add a User to an Alert Rule
+
+You can manage the users who are associated with an alert rule once you know the rule's `rule_id` and the user's `username`.
+
+To add a single user to your alert rule, use the following command.
+```bash
+code42 alert-rules add-user --rule-id <rule-id> -u sean.cassidy@example.com
+```
+
+Alternatively, to add multiple users to your alert rule, fill out the `add` CSV file template, then use the `bulk add` command with the CSV file path.
+```bash
+code42 alert-rules bulk add Users/my_user/bulk-command.csv
+```
+
+You can remove single or multiple users from alert rules similarly using the `remove` and `bulk remove` commands.
+
+
+## Get CSV Template
+
+The following command will generate a CSV template to either add or remove users from multiple alert rules at once.  The csv file will be saved to the current working directory.
+```bash
+code42 alert-rules bulk generate-template [add|remove]
+```
+
+Each of the CSV templates can then be filled out and used with their respective bulk command.
+```bash
+code42 alert-rules bulk [add|remove] Users/my_user/bulk-command.csv
+```
+
+For example, to associate one or more users to one or more alert rules:
+
+1. Generate a CSV template. Below is the command for generating a template to add users to an alert rules. Once generated, the CSV file is saved to your current working directory.
+
+```bash
+code42 alert-rules bulk generate-template add
+```
+
+2. Use the CSV template to enter each trusted activity's information.
+   The `rule_id` and `username` fields are required. The `rule_id` refers to the ID number of the alert rule to add the users too, and the `username` refers to that of the user to add to the alert rule.
+
+3. Save the CSV file.
```

Comment:
I don't think this example section really adds anything useful, it seems mostly a duplicate of the stuff above

Reply:
I wasn't sure either, I took this section from the `detection-lists` user guide, which is the only one that currently goes into the bulk commands.  Me and Annie briefly discussed the best ways to consistently lay them out, but I don't think we ever made a strong decision.